src/safariservices.cs

```diff
@@ -171,6 +176,15 @@ interface SFSafariExtensionHandling
 
 		[Export ("popoverViewController")]
 		SFSafariExtensionViewController PopoverViewController { get; }
+
+		[Mac (10, 12, 4)]
+		[Async (ResultTypeName="SFValidationResult")]
+		[Export ("validateContextMenuItemWithCommand:inPage:userInfo:validationHandler:")]
+		void ValidateContextMenuItem (string command, SFSafariPage page, [NullAllowed] NSDictionary<NSString, NSObject> userInfo, Action<bool, NSString> validationHandler);
```

Comment:
@timrisi you right, it's still needed :) However the _current_ generated code is

```
public class SFValidationResult {
	// constructors
	public SFValidationResult (bool arg1, Foundation.NSString arg2);
	// properties
	public bool Arg1 { get; set; }
	public Foundation.NSString Arg2 { get; set; }
}
```

By using a delegate the properties will be named based on the parameters.